packages/example-next-13-with-pages/messages/de.json

```
@@ -0,0 +1,14 @@
+{
+  "Index": {
+    "title": "Start",
+    "description": "Das ist die Startseite.",
+    "navigateToAbout": "Zu Über uns navigieren"
+  },
+  "About": {
+    "title": "Über",
+    "description": "Hier geht es über uns."
```

Maybe a bit less colloquial:

```suggestion
    "description": "Das ist die Über-uns-Seite."
```